interpreter/runtimecost.go

```diff
@@ -38,6 +38,12 @@ type ActualCostEstimator interface {
 func CostObserver(tracker *CostTracker) EvalObserver {
 	observer := func(id int64, programStep any, val ref.Val) {
 		switch t := programStep.(type) {
+		case *evalTestOnly:
+			// Protobuf has() checks do not incur runtime cost due to not using Eval()
```

Comment:
The `evalTestOnly` instruction should contain an `InterpretableAttribute` and a qualifier. The `InterpretableAttribute` in the operand will have had it's cost tracked already, but the presence test check should probably still count the same as `common.SelectAndIdentCost` and not differentiate based on whether it's a proto field or not.

Reply:
Gotcha, I've used `common.SelectAndIdentCost` and updated the relevant test cases, should be fixed now